parcels/field.py

```diff
@@ -1174,6 +1174,14 @@ def __enter__(self):
     def __exit__(self, type, value, traceback):
         self.dataset.close()
 
+    def parse_name(self, dimensions, variable):
+        name = dimensions['data'] if 'data' in dimensions else variable
+        if isinstance(name, list):
+            for nm in name:
+                if hasattr(self.dataset, nm):
+                    name = nm
```

Comment:
You would like to put a `break` statement here. Such that if you find the name in your list, you stop searching (then it will also make a priority: first name in the list = preferred name

Reply:
We should add a piece of code like this 

`if isinstance(name, list): raise ...`

such that if nothing was found we throw an error